grails-datastore-gorm-test/src/test/groovy/grails/gorm/services/multitenancy/schema/SchemaPerTenantSpec.groovy

```diff
@@ -28,6 +29,8 @@ class SchemaPerTenantSpec extends Specification {
     def setup() {
         System.setProperty(SystemPropertyTenantResolver.PROPERTY_NAME, "")
     }
+
+    @Ignore("java.lang.IllegalStateException: Either class [grails.gorm.services.multitenancy.schema.Book] is not a domain class or GORM has not been initialized correctly or has already been shutdown. Ensure GORM is loaded and configured correctly before calling any methods on a GORM entity.")
```

Comment:
We can use @PendingFeature(reason = '...') here instead of @Ignore.

Reply:
Thanks, I've pushed this change